src/main/drivers/bus_spi_hal.c

```diff
@@ -295,10 +295,10 @@ static bool spiBusReadBuffer(const busDevice_t *bus, uint8_t *out, int len)
 }
 
 // return uint8_t value or -1 when failure
```

Comment:
I think this comment (and another similar one in the file) should just be removed. It's inaccurate and misleading.